packages/asset-swapper/src/noop_samplers/AaveV2Sampler.ts

```diff
@@ -0,0 +1,57 @@
+import { BigNumber } from '@0x/utils';
+
+import { ZERO_AMOUNT } from '../constants';
+export interface AaveInfo {
+    lendingPool: string;
+    aToken: string;
+    underlyingToken: string;
+}
+// tslint:disable-next-line:no-unnecessary-class
+export class AaveV2Sampler {
+    public static sampleSellsFromAaveV2(
+        aaveInfo: AaveInfo,
+        takerToken: string,
+        makerToken: string,
+        takerTokenAmounts: BigNumber[],
+    ): BigNumber[] {
+        // Deposit/Withdrawal underlying <-> aToken is always 1:1
```

Comment:
I think I've asked before. So they inflate everyone's balance of aTokens as interest accrues?